Registrar.Database/Interfaces/IRegiSettingStore.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+using Registrar.Models.Request;
+using Registrar.Models.Response;
+using System.Collections.Generic;
+
+namespace Registrar.Database.Interfaces
+{
+    public interface IRegiSettingStore
+    {
+        Task<SingleRegiSettingResponse> UpdateSetting(UpdateRegiSetting settingName);
```

Comment:
It's not pointless. It's an application model, and can be adjusted to have specific methods or utilities to manipulate it throughout transient use. Response models dictate what is sent to the user/api. It doesn't make sense for a database to return a response model.